paper_eval.py

```diff
@@ -38,11 +42,22 @@
     "Leave empty to use the directory you are running this file from.",
 )
 argparser.add_argument("--seed", type=int, default=DEFAULT_SEED)
-argparser.add_argument("--num_workers", type=int, default=4)
+argparser.add_argument("--num_workers", type=int, default=64)
 argparser.add_argument("--wandb", dest="wandb", action="store_true")
 argparser.add_argument("--wandb_org", type=str, default="nasa-harvest")
 argparser.add_argument("--parquet_file", type=str, default="rawts-monthly_calval.parquet")
-argparser.add_argument("--val_samples_file", type=str, default="cropland_test_split_samples.csv")
+
+# argparser.add_argument("--val_samples_file", type=str, default="cropland_spatial_generalization_test_split_samples.csv")
+
+argparser.add_argument("--presto_model_type", type=str, default="presto-ft-ct")
```

Comment:
Is this argument (`presto_model_type`) only used to name the experiment file? if so can it be given a different name; the current name implies it will somehow affect the model

Reply:
Same for other names which don't affect the functionality, e.g. `compositing_window`. 

Also I think the most up to date `main` hasn't been merged into this, since the 10d compositing is also supported but doesn't seem to be here.

Reply:
good point. addressed that here https://github.com/WorldCereal/presto-worldcereal/commit/1601dbc3c6cb601846c65e3b18914fb46b5946c0